redox/response_flowsheet.go

```diff
@@ -251,15 +270,58 @@ func SetAccountNumberInFlowsheet(order models.NewOrder, flowsheet *models.NewFlo
 }
 
 func SetOrderIdInFlowsheet(order models.NewOrder, flowsheet *models.NewFlowsheet) {
-	if order.Order.ID != "" {
-		extensions := map[string]any{
-			AdditionalIdentifierExtensionId: AdditionalIdentifierExtension{
-				URL: AdditionalIdentifierURI,
-				Identifier: AdditionalIdentifier{
-					Type:  AdditionalIdentifierTypeOrderId,
-					Value: order.Order.ID,
-				},
-			}}
+	if order.Order.ID == "" {
+		return
+	}
+
+	initVisitExtensions(flowsheet)
```

Comment:
There are a lot of skipped null checks in these functions.

There's no logging if a nil is found, and there's returned errors. If something did go wrong, I guess we'd be hoping to pick up on increased 500s?

I trust you've evaluated this and this is the best way forward under the circumstances.

Reply:
I'm not sure I understand your comment. Can you please elaborate?

The reason why we early return in those functions is because we have nothing to add to the flowsheet if the fields are not present in the original message. It's ok for the original message to not have those - they are optional.